ProcessLib/ProcessVariable.h

```diff
@@ -81,8 +81,9 @@ class ProcessVariable
 		for (auto& config : _neumann_bc_configs)
 		{
 			config->initialize(searcher);
-			bcs++ = new NeumannBc<GlobalSetup>(*config,
-			                                   std::forward<Args>(args)...);
+			bcs++ = std::unique_ptr<NeumannBc<GlobalSetup>>{
```

Comment:
For the future: The parameter `GlobalSetup const&` is not used. So maybe it can be removed in the future?